vector/build.gradle

```diff
@@ -97,9 +97,21 @@ android {
         javaMaxHeapSize "2g"
     }
 
+    signingConfigs {
+        // DO NOT USE THIS KEY FOR GOOGLE PLAY RELEASE
+        // TODO STEPS to create a new key and sign the app with the key
```

Comment:
Please refer to your design document for the steps, which then refers to this URL

Reply:
I added another line here.